src/js.cookie.js

```diff
@@ -24,6 +24,13 @@
 		};
 	}
 }(function () {
+	var officialAttributes = {
```

Comment:
I just need the list of official attributes somewhere. And I've found a place that is called just once. There's no other intent :)